test/jit/test_freezing.py

```diff
@@ -1517,3 +1517,36 @@ def test_optimize_freeze_module(self):
         # optimize_frozen_module should be run
         frozen_mod = torch.jit.freeze(torch.jit.script(mod.eval()))
         FileCheck().check_not("batch_norm").run(frozen_mod.graph)
+
+    def test_freeze_remove_dropout(self):
+        class Net(nn.Module):
+            def __init__(self):
+                super(Net, self).__init__()
+                self.conv1 = nn.Conv2d(1, 32, 3, 1)
+                self.conv2 = nn.Conv2d(32, 64, 3, 1)
+                self.dropout1 = nn.Dropout2d(0.25)
+                self.dropout2 = nn.Dropout2d(0.5)
+                self.fc1 = nn.Linear(9216, 128)
+                self.fc2 = nn.Linear(128, 10)
+
+            def forward(self, x):
+                x = self.conv1(x)
+                x = nn.functional.relu(x)
+                x = self.conv2(x)
+                x = nn.functional.max_pool2d(x, 2)
+                x = self.dropout1(x)
+                x = torch.flatten(x, 1)
+                x = self.fc1(x)
+                x = nn.functional.relu(x)
+                x = self.dropout2(x)
+                x = self.fc2(x)
+                output = nn.functional.log_softmax(x, dim=1)
+                return output
+
+        mod = torch.jit.script(Net())
+        # set optimize to False here, by default freezing runs optimize_frozen_module
+        frozen_mod = torch.jit.freeze(torch.jit.script(mod.eval()), optimize=False)
```

Comment:
While we're both pretty sure that it removing dropout doesn't effect results, it might be good to test it by comparing the results of the eager module with the frozen one after optimization